src/index.html

```diff
@@ -22,6 +23,11 @@
 
         <main class="container">
         <!-- Add your site or application content here -->
+
+          <header>
+            <button id="fav-button"><i class="fa fa-heart-o"></i> Favorite Shop</button>
+          </header>
+
```

Comment:
Like we discussed in class, this is probably best placed in one of the _other_ HTML files...